README.md

```diff
@@ -1,2 +1,22 @@
-# Untitled-Engine
-An engine that may or may not be titled
+![README_Banner](https://raw.githubusercontent.com/Untitled-Games/Branding/Untitled-Engine/Untitled-Engine/UE_Full.png)
+# About
+An engine that may or may not be titled!
+
+# Getting Started
+## Cloning the project
+If you wish to clone this project, make sure to clone all the submodules too:
+```
+git clone --recursive https://github.com/SzyJ/midori_engine
+```
+... or ...
+```
+git clone https://github.com/Untitled-Games/Untitled-Engine
+cd Untitled-Engine
+git submodule init
+git submodule update
+```
+
+## Premake
+This project uses [premake](https://github.com/premake/premake-core) to generate desired project files.
+
+```generate_vs2017.bat``` or ```generate_vs2019.bat``` scripts can be used to generate a Visual Studio solution.
```

Comment:
Rename generation scripts to camel case and reference updated names here.

Reply:
Done!